lib/SodiumCrypto.js

```diff
@@ -590,4 +590,20 @@ export default class SodiumCrypto extends Crypto {
     // Returns a File object
     return decryptedBlob
   }
+
+  /**
+   * Helper function for tsv1 which wraps sodium's crypto_generichash function, uses BLAKE2b. https://libsodium.gitbook.io/doc/hashing/generic_hashing
+   */
+  genericHash(rawString) {
+    return sodium.crypto_generichash(sodium.crypto_generichash_BYTES, rawString)
```

Comment:
Does it make sense to b64 encode this hash before returning as well as part of the tozny abstraction over the hashing method?

Reply:
My preference would be to deal with bytes, then b64 encode/decode with our methods when you need to; but I see now most of the other methods in this lib return b64 so I'll follow that convention for consistency 👍